data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3433 @@
+			choice
+				`(I am involved, may as well know all about it. Wake her).`
+				`(Ignorance is bliss).`
+					goto bliss
+			`You were readying to slap her a bit to bring her back, but he stops you, "I will do this, but then nothing else." He goes to take a bucket of water to wake her up. She seems to recognize the hopelessness of her situation, and is willing to bargain, "I will tell you the truth, but then you will let me go and we will never see each others again. But let me be clear, I will not speak, under any circumstances other than those. I also seriously doubt your competences as a torturer so spare me any threats."`
+			choice
+				`(This is it. I want to know all).`
+				`(Ignorance is bliss).`
+					goto bliss
+			`She takes a deep breath, "An important organization wants these gold bars you have on boards. They are not just normal gold. If you were to sell them to me I could give you a good price, and you would not be bothered by this again. 1 million per bar. You see, something is within these bars, something of the highest importance, and they want all of them. They will not stop until they have all of them."`
+			choice
+				`(Sell the bars).`
+				`(Keep them, and drop her on the planet).`
+					goto bliss
+				`(Keep her for further interrogating).`
+					goto escape
+			`She nods, "this is the right choice you are making, captain <last>. You will never be bothered by any of this again, that I promise. I will take these 5 piles, and any more will be sold little by little as you go on any world. If you sell them to any normal outfitters we will easily get our hands on it, so it is no problem to us but you would only be payed the price of brute gold for it."`
+			action
+				outfit "Gold Bars" -5
+				payment 5000000
+				set "sold bars"
+			``
+				decline
+			label escape
+			`You tighten her hold, and tell your marine to keep a watch on her until you can find someone else of trust, but when you come back later, they are both gone. There is a little word on the spot where some gold bars where. It is written by your marine, and says, "<first>. I am very sorry to break your trust, but trust me, this is for your own good. She would have been killed if he returned without the gold bars, so I let her take a few. You should quit this whilst you still can."`
+
+mission "Unfettered: Bar Selling"
+	invisible
+	landing
+	repeat
+	to offer
+		has "sold bars"
+	on offer
+		outfit "Gold Bars" -1
+		payment 1000000
```

Comment:
What is the purpose of this? It seems the bars are removed in the mission above if they are sold.

Reply:
There is no command to 'remove all outfits of this kind'
Thus I have to do it this way, one by one